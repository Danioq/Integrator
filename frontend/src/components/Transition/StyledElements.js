import styled from 'styled-components'


export let Sign = styled.p`
  text-align: center;
  position: absolute;
  top: 50%;
  left: 0;
  right: 0;
  transform: translate(0, -50%);
  font-size: 3em;
  font-weight: bold;
  color: #03c008;
`;