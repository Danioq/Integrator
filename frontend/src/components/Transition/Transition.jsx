import React from 'react'
import {Sign} from './StyledElements'

export default function Transition() {
    return <Sign>≈Åadowanie danych...</Sign>
}