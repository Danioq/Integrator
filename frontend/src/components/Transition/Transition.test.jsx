import Transition from './Transition';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Transition component suite', () => {
  test('Render test', () => {
    let component = renderer.create(<Transition />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});