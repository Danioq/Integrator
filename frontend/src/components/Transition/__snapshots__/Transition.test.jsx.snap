// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transition component suite Render test 1`] = `
<p
  className="sc-AxjAm gPICbf"
>
  ≈Åadowanie danych...
</p>
`;
